fulp_modules/Pet_Objectives/objective_items.dm

```diff
@@ -0,0 +1,109 @@
+/datum/objective_item/steal/iandog
```

Comment:
these should be their own subtype instead of copypasting the special completion multiple times, makes it easier to do in the future as well

Reply:
What do you mean by being their own subtype?